navigation/design.md

```diff
@@ -0,0 +1,35 @@
+# Navigation
```

Comment:
When navigating from graph to workloads/services, then the breadcrumbs should appear, and when navigating from services to graph it should dissapear?
Which is another way of saying "graph should not have breadcrumbs, correct?" :)

Reply:
How the user should be able to navigate from a single entity page to the "plural" page? (from service A back to services, from workload C to workloads page, etc.) back button? via breadcrumb? via left menu only?

Reply:
Per https://issues.jboss.org/browse/KIALI-1262 - if navigating from lets say service (main section Services) to one of the applied rules (In Istio Config section) - how should the breadcrumb look like?
cc @lucasponce

Reply:
@abonas the convention we have in OpenShift, which is what we are recommendation here, is that if you want to view the "plural" page to see all, you would utilize the navigation

Reply:
@serenamarie125 @cshinn so graph should not have breadcrumbs, correct?

Reply:
@abonas that's correct